app/src/main/java/com/practicum/playlistmaker/SearchActivity.kt

```diff
@@ -120,6 +130,64 @@ class SearchActivity : AppCompatActivity() {
         inputEditText.setText(inputValue)
     }
 
+    private fun search() {
+        iTunesService.findSong(inputEditText.text.toString()).enqueue(object : Callback<TracksResponse> {
+            override fun onResponse(
+                call: Call<TracksResponse>,
+                response: Response<TracksResponse>
+            ) {
+                tracks.clear()
+                if (response.code() == 200) {
```

Comment:
```suggestion
                if (response.isSuccessfull) {
```
Лучше воспользоваться этим свойством для проверки того, был ли ответ успешно получен